client/landing/gutenboarding/onboarding-block/design-selector/design-card.tsx

```diff
@@ -19,8 +19,15 @@ interface Props {
 	onClick: MouseEventHandler< HTMLButtonElement >;
 	style?: CSSProperties;
 	dialogId: string;
+	tabIndex: number;
 }
-const DesignCard: FunctionComponent< Props > = ( { design, dialogId, onClick, style } ) => (
+const DesignCard: FunctionComponent< Props > = ( {
+	design,
+	dialogId,
+	onClick,
+	style,
+	tabIndex = 0,
```

Comment:
Is `tabindex="0"` on an interactive element (like the `button` we'll render here) redundant? If so, we can omit this and don't need to worry about setting the attribute at all.

Reply:
The parent component is deciding whether to set `tabIndex` to `0` or `-1` based on the visibility of the page selector that's why it's here.

Reply:
This is a minor point, but I don't think I was clear to I'll elaborate. This is an opportunity for the parent to pass _any_ `tabIndex` (not just `0` or `1`), which will be passed to the `Card`.

My question was about the default `0`. If the `tabIndex` is not provided by the parent, `<button tabindex="0"/>` will have the same tabbing behavior as `<button />` without the attribute, right? In that case, we don't need to pass `0` at all:

```suggestion
	tabIndex,
```